_episodes/04-use-license.md

```diff
@@ -5,7 +5,185 @@ exercises: 0
+* In particular, the code that you write is copyrighted by default in the most restrictive way. This implies that nobody, except you, can use, modify (even to fix bugs!), create derivative works, distibute it.
+* If you infringe on someone's copyright you may  be in the situation of having to pay reasonable damages.
+* "Fair rules" - parody, criticism, quotation - are an exception to these rights. For example, you can publish a study comparing the performance or the accuracy of several different programs, without infringing the copyright of the programs.
+
+> ### NOTE: Copyright vs Patent vs Trademarks
+>
+> **Copyright** protects the creative expression of ideas, **patents** protect novel technological inventions, **trademarks** protect ownership.
+{: .callout}
+
+> ### Challenge
+>
+> For each one of the three scenarios below, answer yes or no to the question.
+> 
+> 1. **Scenario:** you receive a sticker and your friends want a copy.
+> 
+>    **Question:** can you make a copy of the sticker and sell the copy to your friends?
+> 2. **Scenario:** you receive a script from your supervisor asking you to modify it to analyse a different type of input data.
+>
+>    **Question:** are you allowed to do change the script?
+> 3. **Scenario:** you download a collection of photos from a shared photo website?
+>
+>    **Question:** are you allowed to use the photos that you downloaded for your machine learning algorithm?
+{: .challenge}
+
+### What a licence does 
+
+A licence is an official permission to do, use, or own something. Basically, a licence waives parts of a copyright by defining terms and conditions for the usage of a product.   
+A shorthand definition of license is "a promise by the licensor not to sue licensees" if they comply with the licence terms and conditions.   
+
+What an open source licence does?
+
+Authors of code tend to share it expecting that others will use, modify and share it. However, as the default is exclusive copyright, you need to explicitely state that you allow others to use, modify and share your code.   
+The way to do it, is by assigning an open source licence to the code that you share with others, even if you are not publishing it but simply sharing it, e.g., by email.  
+Notice that "if you do not assign an open source license to your project, everybody who contributes to it becomes an exclusive copyright holder of their work. That means 'nobody' can use, copy, distribute, or modify their contributions – includes yourself." [[ref]](https://opensource.guide/legal/)   
+
+Also, you should be aware of whether other people's code, that you want to reuse or include in your project, has a licence and what type of licence it has. If you use, modify, or distribute a non licenced code, you are potentially liable to be sued by the code's owner.
```

Comment:
"licensed" should always be spelled with an "s", even if you're using the British English spelling of "licence"